### PR TITLE
adds --open / -o option to screenshot command

### PR DESCRIPTION
**Pull request by totolicious**
Adds --open / -o option, which opens the screenshot using the native picture viewer

### CODE REVIEW

File: packages/sdk-cli/src/commands/screenshot.test.ts [hunk 1]
```diff
@@ -10,6 +11,10 @@ import commandTestHarness from '../testUtils/commandTestHarness';
 import { homedir } from 'os';
 
 jest.mock('../models/captureScreenshot');
+jest.mock('open', () => ({
+  __esModule: true,
```
Comment (Hexxeh) [inline packages/sdk-cli/src/commands/screenshot.test.ts]: Never seen this done before in a mock, why is it required?

Why does the `jest.mock` auto-mock not work?

**Review by Hexxeh — APPROVED** [review]

File: packages/sdk-cli/src/commands/screenshot.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import * as path from 'path';
 import dateformat from 'dateformat';
 import untildify from 'untildify';
 import vorpal from 'vorpal';
+import { default as openScreenshot } from 'open';
```
Comment (npclaudiu) [inline packages/sdk-cli/src/commands/screenshot.ts]: I think it's better to import this as `open`, as it does nothing specific to screenshots.

File: packages/sdk-cli/src/commands/screenshot.test.ts [hunk 3]
```diff
@@ -71,6 +76,12 @@ it('logs the location of the file to the console', async () => {
   expect(mockUIRedraw.done).toBeCalled();
 });
 
+it('opens the screenshot', async () => {
+  const destPath = path.resolve('Screenshot 2018-03-04 at 15.06.07.png');
+  await expectDestPath(destPath, undefined, true);
```
Comment (npclaudiu) [inline packages/sdk-cli/src/commands/screenshot.test.ts]: nit: Maybe rename `expectDestPath` to something like `expectCaptureScreenshotCall`?

**Review by npclaudiu — APPROVED** [review]

File: packages/sdk-cli/src/commands/screenshot.ts (hunk 2 above)
Comment (totolicious) [reply packages/sdk-cli/src/commands/screenshot.ts]: This allowed me to avoid variable shadowing. (See line 20)

Comment (totolicious) [reply packages/sdk-cli/src/commands/screenshot.ts]: Thanks

File: packages/sdk-cli/src/commands/screenshot.test.ts (hunk 1 above)
Comment (totolicious) [reply packages/sdk-cli/src/commands/screenshot.test.ts]: Thanks!
Not sure why I wrote it that way

File: packages/sdk-cli/src/commands/screenshot.test.ts (hunk 3 above)
Comment (totolicious) [reply packages/sdk-cli/src/commands/screenshot.test.ts]: Thanks, I renamed it

**Pull request merged**